runtime/runtime/src/actions.rs

```diff
@@ -316,7 +265,6 @@ pub(crate) fn action_create_account(
         locked: 0,
         code_hash: CryptoHash::default(),
         storage_usage: fee_config.storage_usage_config.num_bytes_account,
```

Comment:
This is different from what `Account::new` does. The value might need to be updated, since you've removed the `storage_paid_at` below.

Reply:
Account:new is only used in tests from what I see... Seems a bit weird